R/CreatePackageVignette.R

```diff
@@ -0,0 +1,101 @@
+#' @title pkgnet Report as Vignette
+#' @name CreatePackageVignette
+#' @description Create pkgnet package report as an Rmarkdown vignette. This
+#'    vignette is able to be rendered with the
+#'    \code{\link[knitr:vignette_engines]{knitr::rmarkdown}} vignette engine
+#'    into HTML vignettes upon package building. It is also compatible with
+#'    \code{\link[pkgdown:build_articles]{pkgdown}} sites.
+#' @param pkg_name (string) name of a package
+#' @param pkg_path (string) The path to the package repository. If given, coverage
+#'                 will be calculated for each function. \code{pkg_path} can be an
+#'                 absolute or relative path.
+#' @param pkg_reporters (list) a list of initialized package reporters
+#' @param vignette_path (string) The path and filename of the output vignette
+#'    file. The default assumes your working directory is the package root.
+#' @importFrom rlang enexpr
+#' @importFrom assertthat assert_that is.string is.readable
```

Comment:
you need `as.dir` here too

Reply:
I must've taken this out because I don't see `assertthat::is.dir` in my code anymore.